okta/framework/Serializer.py

```diff
@@ -5,10 +5,11 @@
 
 class Serializer(JSONEncoder):
     def default(self, obj):
-
         if isinstance(obj, datetime):
             return obj.strftime('dt(%Y-%m-%dT%H:%M:%SZ)')
         elif isinstance(obj, object):
-            return Utils.remove_nulls(obj.__dict__)
+        	no_nulls = Utils.remove_nulls(obj.__dict__)
+        	formatted = Utils.reformat_obj(obj, no_nulls)
```

Comment:
Cool, maybe name it something like `replace_alt_names` instead? I'm not sure what `reformat_obj` does.